From 3aec2e590280635e6855d425d349a2809a31477e Mon Sep 17 00:00:00 2001
From: Reuben Kuhnert <reubenkuhnert@ReubenKuhnert-MBP.local>
Date: Wed, 8 Jun 2016 13:35:50 -0500
Subject: [PATCH 0618/1164] CLOUDERA-BUILD: CDH-41166: Fix missing dependency
 causing test failures.

Add log4j-api dependency to fix broken builds.

Change-Id: I2e521f4b9d3394fbcd4934193d5b1ab2b6179545
---
 testutils/ptest2/pom.xml |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/testutils/ptest2/pom.xml b/testutils/ptest2/pom.xml
index 211678e..2ec89c7 100644
--- a/testutils/ptest2/pom.xml
+++ b/testutils/ptest2/pom.xml
@@ -114,6 +114,11 @@ limitations under the License.
       <version>1.7.3</version>
     </dependency>
     <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+      <version>2.2</version>
+    </dependency>
+    <dependency>
       <groupId>org.apache.velocity</groupId>
       <artifactId>velocity</artifactId>
       <version>1.7</version>
-- 
1.7.9.5

